tutorials/forward/25_automated_coreg.py

```diff
@@ -23,17 +23,17 @@
 # License: BSD-3-Clause
 
 import numpy as np
+from pathlib import Path
 
 import mne
 from mne.coreg import Coregistration
 from mne.io import read_info
 
-
-data_path = mne.datasets.sample.data_path()
-subjects_dir = data_path + '/subjects'
+data_path = Path(mne.datasets.sample.data_path())
```

Comment:
This is already a `Path` object, so no need to import and convert.

Reply:
That was an attempt to make the use of the `Path` object more obvious. Maybe a comment instead of the redundant import/conversion?